app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -145,6 +147,43 @@ class PublicHealthHeader extends React.Component {
     );
   }
 
+  renderWarningModal() {
+    return (
+      <Modal
+        centered
+        dialogClassName="modal-import"
+        className="import-warning-modal-container"
+        scrollable="true"
+        backdrop="static"
+        size="lg"
+        show={this.state.showWarningModal}
+        onHide={async () => {
+          this.setState({ showWarningModal: false, importType: null, importData: null });
+        }}>
+        <Modal.Header closeButton>
+          <Modal.Title className="h5">{this.state.importType === 'epix' ? 'Import Epi-X' : 'Import Sara Alert Format'} (warning)</Modal.Title>
```

Comment:
Minor, but is there a reason the warning is in parenthesis?  I think it would look clearer if it was just `Warning`

Reply:
The warnings in the body also have the warning label, which seems redundant.  I think we should either remove the warning text from the header or remove it from the warning list.  Leaning towards the latter

Reply:
Thanks @sgober! It definitely is redundant to have warning repeated. Updated per your suggestion [here](https://github.com/SaraAlert/SaraAlert/pull/1168/commits/79ee91d0cf33ae4029566bd159f74ba02ae94619).